application/forms/Config/ConfigForm.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+// Icinga Web JIRA Integration | (c) 2023 Icinga GmbH | GPLv2
+
+namespace Icinga\Module\Jira\Forms\Config;
+
+use Icinga\Application\Config;
+use Icinga\Module\Jira\RestApi;
+use Icinga\Web\Notification;
+use ipl\Validator\CallbackValidator;
+use ipl\Web\Compat\CompatForm;
+
+class ConfigForm extends CompatForm
+{
+    /**
```

Comment:
Please change the comment to one line.
`/** @var string Deployment type of the Jira Software cloud or server */`